apps/webcert/src/store/certificate/certificateMiddleware.ts

```diff
@@ -960,10 +960,12 @@ const handleAutoSaveCertificateError: Middleware<Dispatch> =
   ({ dispatch }: MiddlewareAPI<AppDispatch, RootState>) =>
   () =>
   (action: AnyAction): void => {
-    if (action.payload.error.errorCode === 'UNKNOWN_INTERNAL_PROBLEM') {
-      dispatch(throwError(createConcurrencyErrorRequestFromApiError(action.payload.error)))
-    } else {
-      dispatch(throwError(createErrorRequestFromApiError(action.payload.error)))
+    if (action.payload.error) {
```

Comment:
When a request failed due to already being loaded, it will not contain a `.error` and it should be ignored.